list.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+
+  def each
+    i = 0
+    while i < @list.length
+      yield @list[i]
+      i += 1
+    end
+  end
```

Comment:
- [ ] [OPTIONAL] each function show return list like so;
```
  def each
    i = 0
    while i < @list.length
      yield @list[i]
      i += 1
    end
    @list
  end
```

Reply:
I have done the optional changes.